controller/controller.go

```diff
@@ -21,19 +21,19 @@ var (
 )
 
 // Handle creates the router and starts the server
-func Handle(securityConfig *security.Config, webConfig *web.Config, uiConfig *ui.Config, enableMetrics bool) {
-	var router http.Handler = handler.CreateRouter(ui.StaticFolder, securityConfig, uiConfig, enableMetrics)
+func Handle(config *config.Config) {
```

Comment:
Could you pass the parameter `config` as `cfg` so that it doesn't collide with the `config` package?

Reply:
Resolved by #307